test/functional/apps/console/_misc_console_behavior.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const retry = getService('retry');
+  const log = getService('log');
+  const browser = getService('browser');
+  const PageObjects = getPageObjects(['common', 'console', 'header']);
+
+  describe('misc console behavior', function testMiscConsoleBehavior() {
+    this.tags('includeFirefox');
+    before(async () => {
+      log.debug('navigateTo console');
+      await browser.setWindowSize(1200, 800);
+      await PageObjects.common.navigateToApp('console');
+      // Ensure that the text area can be interacted with
+      await PageObjects.console.collapseHelp();
+    });
+
+    beforeEach(async () => {
+      await PageObjects.console.clearTextArea();
+    });
+
+    describe('collapsible JSON blocks', () => {
+      it('should collapse and expand JSON blocks', async () => {
+        const blockNumber = 1;
+        await PageObjects.console.enterRequest(
+          'GET _search\n{\n  "query": {\n    "match_all": {}\n'
+        );
+        // Sleep to allow the JSON to be parsed and rendered
+        await PageObjects.common.sleep(1000);
```

Comment:
I know we ran this 200 times through flaky test runner, but could we run it 1000 times to make sure that this sleep works properly?

Reply:
The total job count is limited to 500 in flaky test runner, so I am running it 497 times plus the builds. I'll leave a comment in the PR once it is all successful https://buildkite.com/elastic/kibana-flaky-test-suite-runner/builds/1284